server/utils/risksAndNeeds/healthUtils.test.ts

```diff
@@ -0,0 +1,36 @@
+import HealthUtils from './healthUtils'
+import { healthFactory } from '../../testutils/factories'
+
+describe('HealthUtils', () => {
+  describe('healthSummaryListRows', () => {
+    it('formats the health data in the appropriate format for passing to a GOV.UK summary list Nunjucks macro', () => {
+      const health = healthFactory.build({
+        anyHealthConditions: true,
+        description: 'Some health conditions',
+      })
```

Comment:
Could just use an explicitly-typed object here (and below) instead of using the factory, given all the keys are present